build.gradle

```
@@ -13,8 +13,8 @@ plugins {
 	id "org.quiltmc.gradle.licenser" version "1.1.1"
 	id "org.ajoberstar.grgit" version "3.1.0"
 	id "com.matthewprenger.cursegradle" version "1.4.0"
+	id "me.modmuss50.remotesign" version "0.3.0" apply false
 	id "com.modrinth.minotaur" version "2.2.0"
```

Don't shuffle the order here, it'll confuse Git diffs
```suggestion
	id "com.modrinth.minotaur" version "2.2.0"
	id "me.modmuss50.remotesign" version "0.3.0" apply false
```